src/theory/bags/solver_state.h

```diff
@@ -31,24 +31,43 @@ class SolverState : public TheoryState
  public:
   SolverState(context::Context* c, context::UserContext* u, Valuation val);
 
-  void registerClass(TNode n);
+  void registerBag(TNode n);
```

Comment:
Can you add some documentation here? When is this method called? What is the effect (is n added to some set of this class i assume?).